### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
Added product listing page.

site link: https://ym-store-products.netlify.app/store-pages/product-page/product-page.html

### CODE REVIEW

File: store-pages/product-page/product-page.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    grid-area: header;
+    position: sticky;
+    top: 0;
+    z-index: 10;
+}
+
+.search-bar-border {
+    border: 2px solid var(--color-white);
+    padding: 0.2rem 0.6rem;
+    border-radius: 0.5rem;
```
Comment (omkarpatke) [inline store-pages/product-page/product-page.css]: you can use px here as Vikram thought if the value is less than 1rem use px.

File: store-pages/product-page/product-page.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+/* Main Section */
+.main {
+    grid-area: main;
+    margin: 0 1rem;
+}
+
+.product-container {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0.5rem;
```
Comment (omkarpatke) [inline store-pages/product-page/product-page.css]: here as well

**Review by omkarpatke — CHANGES_REQUESTED** [review]

**Comment by omkarpatke** [comment]
bug
here both are selecting so work on that.
in your card add more gap so it looks better.
otherwise, all looks good.

**Comment by Sahil25061999** [comment]
didn't find anything apart from what @omkarpatke  has mentioned. The rest of  the code looks good.

**Pull request merged**